install/installer/pkg/components/iam/constants.go

```diff
@@ -7,10 +7,12 @@ package iam
 const (
 	Component = "iam"
 
-	GRPCPortName      = "grpc"
-	GRPCContainerPort = 9001
-	GRPCServicePort   = 9001
-	HTTPContainerPort = 9002
-	HTTPServicePort   = 9002
-	HTTPPortName      = "http"
+	GRPCPortName               = "grpc"
+	GRPCContainerPort          = 9001
+	GRPCServicePort            = 9001
+	HTTPContainerPort          = 9002
+	HTTPServicePort            = 9002
+	HTTPPortName               = "http"
+	oidcClientsSecretMountPath = "oidc-clients-secret"
+	oidcClientsSecretFilename  = "config.json"
```

Comment:
I thought we would use dynamic (i.e. DB persisted) configuration for oidc clients. Is this something else?

Reply:
Ah, I think I get it: this is our secret that we are sending to the provider in order to authenticate gitpod as a client.

Reply:
DB models, persistence, and gRPC services are tackled separately, see https://github.com/gitpod-io/gitpod/issues/15150

This is a 🛹  to decouple the service.